groups_and_permissions_fr.md

```diff
@@ -173,4 +173,4 @@ La webadmin émettra un avertissement si vous définissez une permission qui est
 Notes pour les packageurs d'applications
 ------------------------
 
-[Voir la version anglaise de cette page](groups_and_permissions_en).
+[Voir la version anglaise de cette page](groups_and_permissions).
```

Comment:
Eeeeh unfortunately that one is intentional because it's meant to point to the english version explicitly ... but I understand that the tests report it (I wrote them ;P) . I think in fact it's bad designed to have the english version of this page include a huge section only meant for packagers ... we should create a dedicated page for the maintainer infos which would solve the issue...

Reply:
@alexAubin : no worries, and thanks for the explanation. I thus understand I should leave this file unchanged, is that right?
As for the better/full solution, you are advising we split out the packager-only instructions out of the english page and move it to a dedicated page, then translated in both English and French, is that it?

Reply:
Yes, but no need to translate the page .... that's a documentation meant for packagers and most of them are used to deal with english except of somebody really is motivated ... except if somebody really is motivated about it :/

Reply:
Ok, thanks for the feedback, I have reverted the change on that file and squashed everything so that it looks cleaner.